assets/js/googlesitekit/data/create-error-store.test.js

```diff
@@ -0,0 +1,221 @@
+			it( 'requires the error param', () => {
+				expect( () => {
+					dispatch.receiveError();
+				} ).toThrow( 'error is required.' );
+			} );
+
+			it( 'receives and sets value', () => {
+				dispatch.receiveError( ERROR_SAMPLE.error, ERROR_SAMPLE.baseName, ERROR_SAMPLE.args );
+				expect( store.getState().errors[ ERROR_SAMPLE.baseName ] ).toMatchObject( ERROR_SAMPLE.error );
+			} );
+		} );
+
+		describe( 'clearError', () => {
+			it( 'removes an error', () => {
+				dispatch.receiveError( ERROR_SAMPLE.error, ERROR_SAMPLE.baseName, ERROR_SAMPLE.args );
+				dispatch.receiveError( ERROR_SAMPLE_2.error, ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+				dispatch.clearError( ERROR_SAMPLE.baseName );
+
+				const errorObj = {};
+				errorObj[ ERROR_SAMPLE_2.baseName ] = ERROR_SAMPLE_2.error;
+				expect( store.getState().errors ).toMatchObject( errorObj );
+			} );
+		} );
+	} );
+
+	describe( 'selectors', () => {
+		describe( 'getErrorForSelector', () => {
+			it( 'requires a selector param', () => {
+				expect( () => {
+					select.getErrorForSelector();
+				} ).toThrow( 'selectorName is required.' );
+			} );
+
+			it( 'returns undefined when error does not exist', () => {
+				dispatch.receiveError( ERROR_SAMPLE.error, ERROR_SAMPLE.baseName, ERROR_SAMPLE.args );
+				const selectedError = select.getErrorForSelector( ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+				expect( selectedError ).toBeUndefined();
+			} );
+
+			it( 'returns the required error', () => {
+				dispatch.receiveError( ERROR_SAMPLE.error, ERROR_SAMPLE.baseName, ERROR_SAMPLE.args );
+				dispatch.receiveError( ERROR_SAMPLE_2.error, ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+				const selectedError = select.getErrorForSelector( ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+
+				expect( selectedError ).toMatchObject( ERROR_SAMPLE_2.error );
+			} );
+		} );
+
+		describe( 'getErrorForAction', () => {
+			it( 'requires a selector param', () => {
+				expect( () => {
+					select.getErrorForAction();
+				} ).toThrow( 'actionName is required.' );
+			} );
+
+			it( 'returns undefined when error does not exist', () => {
+				dispatch.receiveError( ERROR_SAMPLE.error, ERROR_SAMPLE.baseName, ERROR_SAMPLE.args );
+				const selectedError = select.getErrorForAction( ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+				expect( selectedError ).toBeUndefined();
+			} );
+
+			it( 'returns the specified error', () => {
+				dispatch.receiveError( ERROR_SAMPLE.error, ERROR_SAMPLE.baseName, ERROR_SAMPLE.args );
+				dispatch.receiveError( ERROR_SAMPLE_2.error, ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+				const selectedError = select.getErrorForAction( ERROR_SAMPLE_2.baseName, ERROR_SAMPLE_2.args );
+
+				expect( selectedError ).toMatchObject( ERROR_SAMPLE_2.error );
+			} );
+		} );
+
+		describe( 'getError', () => {
+			it( 'requires a baseName param', () => {
+				select.getError();
+
+				expect( store.getState().error ).toBeUndefined();
+			} );
```

Comment:
Similar to previous tests, this test should assert that the selector throws an error when `args` is provided, but `baseName` is empty.